Spurious test failures in ColumnAdditionByFetchingURLsOperationTests

**Describe the bug**
Sometimes the following test fails:
```
[ERROR] Tests run: 704, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 75.502 s <<< FAILURE! - in TestSuite
[ERROR] testHttpHeaders(com.google.refine.operations.column.ColumnAdditionByFetchingURLsOperationTests)  Time elapsed: 1.751 s  <<< FAILURE!
java.lang.AssertionError: expected [false] but found [true]
	at com.google.refine.operations.column.ColumnAdditionByFetchingURLsOperationTests.runAndWait(ColumnAdditionByFetchingURLsOperationTests.java:133)
	at com.google.refine.operations.column.ColumnAdditionByFetchingURLsOperationTests.testHttpHeaders(ColumnAdditionByFetchingURLsOperationTests.java:266)

```

**To Reproduce**
Run the tests many times? I find it curious that it does not happen on Travis more often, perhaps my machine is just slow.

**Desktop<!--  (please complete the following information)-->:**
 - OS: Linux
 - JRE or JDK Version: OpenJDK 11

**OpenRefine <!--(please complete the following information)-->:**
 - master branch

--- a/main/tests/server/src/com/google/refine/operations/column/ColumnAdditionByFetchingURLsOperationTests.java
+++ b/main/tests/server/src/com/google/refine/operations/column/ColumnAdditionByFetchingURLsOperationTests.java
@@ -263,7 +263,7 @@ public class ColumnAdditionByFetchingURLsOperationTests extends RefineTest {
                 true,
                 headers);
 
-            runAndWait(op, 1000);
+            runAndWait(op, 3000);
 
             RecordedRequest request = server.takeRequest();
             Assert.assertEquals(request.getHeader("user-agent"), userAgentValue);

